app/auth/Signup.tsx

```diff
@@ -1,10 +1,32 @@
 import React from "react";
-import { View, StyleSheet, KeyboardAvoidingView, ScrollView, Platform } from "react-native";
+import { View, StyleSheet, KeyboardAvoidingView, ScrollView, Platform, Keyboard } from "react-native";
 import { router } from "expo-router";
+import { SignupI } from "@/interfaces";
+import { useFormikHook } from "@/hooks";
+import { signupValidationSchema } from "@/utils";
 import { colorPalette, LayoutStyles, Spacing } from "@/styles";
 import { AppButton, AppLogo, AppText, GradientWrapper, TextInput } from "@/components";
 
 const Signup = () => {
+  const validationSchema = signupValidationSchema;
+  const initialValues: SignupI = { name: "", email: "", password: "" };
+
+  const submit = async ({ name, email, password }: SignupI) => {
+    try {
+      Keyboard.dismiss();
+      console.log(name, email, password);
+      router.push("/(tab)/");
+    } catch (err) {
+      console.log("error === ", err);
+    }
+  };
+
+  const { handleChange, handleSubmit, setFieldTouched, errors, touched, values } = useFormikHook(
+    submit,
+    validationSchema,
+    initialValues
+  );
+
   return (
     <GradientWrapper style={LayoutStyles.horizontalSpacing}>
       <KeyboardAvoidingView behavior={Platform.OS === "ios" ? "padding" : "height"} style={styles.container}>
```

Comment:
Remove keyboard avoidview from everywhere. I don't need to comment for this again ever in code!!